RFP-01.md

```diff
@@ -0,0 +1,169 @@
+# RFP-01: Settlement contract wrapper
+
+## Background
+
+The CoW DAO has recently passed [CIP-44: Reduced bonding requirements](https://snapshot.org/#/cow.eth/proposal/0x1b6f1171633ec3d20c4370db37074aa1bd830486d4d0d6c26165915cc42d9412), which allows solvers to participate in the batch auction with reduced bonding requirements. The reduced bonding requirements are as follows:
+
+- $50k in yield-bearing stable coins or ETH
+- 500k COW
+
+The target deposit is:
+
+- $100k in yield-bearing stable coins or ETH
+- 1M COW
+
+## Goal
+
+The goal of this project is to implement a `SubPoolFactory` that allows solvers to participate in the batch auction with reduced bonding requirements. The `SubPoolFactory` will deploy a `SubPool` for each solver, which will be funded to at least the minimum required amount. The `SubPoolFactory` will also provide a wrapper around the `GPv2Settlement` contract that allows solvers to execute their solutions by providing a signature attesting to the solution and their parameters.
+
+### Key Features
+
+- Ultra gas efficient signed settlement function
+- No co-mingling of funds between solvers
+- Easily observable on-chain addresses for MEV Blocker rebates
+- Permissionless access to the batch auction with reduced bonding requirements
+
+### Deliverables
+
+- Contract system for `SubPoolFactory`, `SubPool`, and `SignedSettlement`
+- Comprehensive documentation
+- Open-source code repository
+
+### Interface
+
+```solidity
+interface Auth {
+    /**
+     * Allows `usr` to perform root-level functions
+     */
+    function rely(address usr) external auth;
+    /**
+     * Denies `usr` from being able to perform root-level functions
+     */
+    function deny(address usr) external auth;
+    /**
+     * Determine if `usr` is able to perform root-level functions
+     */
+    function wards(address usr) external view returns (bool);
```

Comment:
```suggestion
    function isOwner(address usr) external view returns (bool);
```

It could all be so simple for contributors with a B1 level in english.